index.html

```diff
@@ -36,6 +36,60 @@
     </div>
     </header>
     <main>
+        <div id="overlay">
+            <div id="modal">
+                <nav class="overlay-nav">
+                    <h3 class="project-title"></h3>
+                    <img id="cancel" src="images/Icon-Cancel.png" alt="close-popup">
```

Comment:
- [ ] Good job so far when we tap/click on the project button the modal appears with information about a specific project :clap: But unfortunately the close (X) button is not working cause the popup doesn't disappear. Please fix it in order to make your modal work as expected :smile:

Reply:
Thanks for telling me but in my machine it is working fine